pub/css/main.css

```diff
@@ -60,27 +60,353 @@ textarea {
 /* ==========================================================================
    Author's custom styles
    ========================================================================== */
+ 
+/* TOP SECTION / HEADER */
+
+header {
+    text-align: center;
+    background-image: url("http://loneswimmer.files.wordpress.com/2012/05/low_force_3_guillamene_ne.jpg");
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: cover;
```

Comment:
You can also roll all these up into `background` although I also find it clearer when using the more explicit properties. Additionally not every property has a short cut with `background`...